contracts/exchangeIssuance/CErc20Storage.sol

```diff
@@ -0,0 +1,24 @@
+/*
+    Copyright 2022 Index Cooperative
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+pragma solidity 0.6.10;
+pragma experimental ABIEncoderV2;
+
+contract CErc20Storage {
```

Comment:
It seems you are using this contract only as interface. In that case I'd declare it as such and move it to the `interfaces` folder. (same for `CompoundLeverageModuleStorage` if you don't remove that one entirely - see comments below)

Reply:
addressed in 2bc43dfd123d9c826c22e1931048af595bb3ebc3